### PR TITLE
add : added image.html and image.css

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
1. added image component
2. added documentation of image component

image

### CODE REVIEW

File: components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        <li><a href="/components/html/rating.html">Rating</a></li>
+        <li><a href="/components/html/toast.html">Toast/Snackbar</a></li>
+        <li><a href="/components/html/card.html">Simplified Grid</a></li>
+    </ul>
+  </div>
+
+    <div class="Component-container">
+
+        <div class="text  mt2" id="image">
+            <h2 class="pb1 h2 comp-title">Images</h2><br>
```
Comment (gauravsinhaweb) [inline components/image/image.html]: there is no use of <br/> here ig

File: components/image/image.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+
+:root{
+    --grey:grey;
+    --white-light:rgb(248, 244, 244);
+
+
```
Comment (gauravsinhaweb) [inline components/image/image.css]: remove extra spacing.

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: components/image/image.html (hunk 1 above)
Comment (JuvitaSaini09) [reply components/image/image.html]: i have written its css into doc.css file ,thankyou for the review

File: components/image/image.css (hunk 2 above)
Comment (JuvitaSaini09) [reply components/image/image.css]: done

**Review by gauravsinhaweb — APPROVED** [review]

File: components/image/image.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        <li><a href="/components/html/toast.html">Toast/Snackbar</a></li>
+        <li><a href="/components/html/card.html">Simplified Grid</a></li>
+    </ul>
+  </div>
+
+    <div class="Component-container">
+
+        <div class="text  mt2" id="image">
+            <h2 class="pb1 h2 comp-title">Images</h2>
+            <p class="pb2">img-parent class is for both the parents of image tags. resp-img class for the responsive
```
Comment (AshutoshM29) [inline components/image/image.html:51]: You should use class names that are understandable to others too

File: components/image/image.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            <iframe id="frame"
+                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=vscode&wt=none&l=auto&width=680&ds=true&dsyoff=61px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%253Cdiv%2520class%253D%2522resp-img%2520img-parent%2522%253E%250A%2520%253Cimg%2520src%253D%2522%252Fdocs%252Fassets%252Favatar.jpg%2522%2520alt%253D%2522responsive%2520image%2522%253E%250A%2520%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520"
+                style=" height: 200px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+        </div>
+
+
+        <br><br>
+        <h2 class="grey mb2">Round Image</h2>
```
Comment (AshutoshM29) [inline components/image/image.html:69]: Again change the class name from "grey" to something else!

File: components/image/image.css [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    --white-light:rgb(248, 244, 244);
+}
+.img-parent{
+    width:31.25rem;
+    border:1px solid var(--grey);
+    padding:1.56rem;
+    width:fit-content;
+    background-color: var(--white-light);
+}
+.img-parent img{
```
Comment (varun1233) [inline components/image/image.css]: If possible try avoiding css chaining, in this case i guess by using only .img-parent also the work can be done

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
Work on the suggestions, the rest looks good

**Review by varun1233 — CHANGES_REQUESTED** [review]

File: components/image/image.html (hunk 3 above)
Comment (JuvitaSaini09) [reply components/image/image.html:51]: okk i will take care of it,thnks fo the review

File: components/image/image.css (hunk 5 above)
Comment (JuvitaSaini09) [reply components/image/image.css]: done,please approve PR

**Review by varun1233 — APPROVED** [review]

File: components/image/image.html (hunk 4 above)
Comment (JuvitaSaini09) [reply components/image/image.html:69]: ok,thnks for the review

**Pull request merged**